client/my-sites/domains/domain-search/index.jsx

```diff
@@ -115,13 +115,18 @@ class DomainSearch extends Component {
 			domainRegistration = updatePrivacyForDomain( domainRegistration, true );
 		}
 
-		addItems( [ domainRegistration ] );
+		addItem( domainRegistration );
 		goToDomainCheckout( suggestion, this.props.selectedSiteSlug );
 	}
 
 	removeDomain( suggestion ) {
 		this.props.recordRemoveDomainButtonClick( suggestion.domain_name );
-		removeDomainFromCart( suggestion );
+		removeItem(
+			domnRegistration( {
```

Comment:
I don't _love_ that this alias is so close to the variable name used in the above function, but it's already there and linters should keep everything safe so 🤷‍♂

Reply:
This can be resolved with a simple variable rename. I'll do a followup PR.

Reply:
I had to rebase and resolve conflicts in this branch, so I pushed the patch here: rename the `domainRegistration` variable to `registration`, which frees up the `domainRegistration` name to import the function without aliasing.